doc/MODDING.md

```diff
@@ -35,7 +35,7 @@ Here is a full list of supported values for MOD_INFO:
 [
   {
     "type": "MOD_INFO",                             // Mandatory, you're making one of these!
-    "id": "Mod_ID",                                 // Mandatory, unique id for your mod. You should not use the same id as any other mod.
+    "id": "Mod_ID",                                 // Mandatory, unique id for your mod. You should not use the same id as any other mod.  Cannot contain the '#' character.
```

Comment:
The mod compatibility feature depends on folder names, so to be compatible with that, it really cannot use any characters that filesystems (the game supports) can't handle.